src/MathContainer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  Switch,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Home from './pages/home';
+import Calculator from './pages/Calculator';
+import Quote from './pages/Quote';
+import NotMatch from './pages/NotMatch';
+
+const MathContainer = () => (
+  <>
+    <Navbar />
+    <Switch>
+      <Route exact path="/">
+        <Home />
+      </Route>
+      <Route exact path="/calculator/">
+        <Calculator />
+      </Route>
+      <Route exact path="/about/">
```

Comment:
- [ ] [OPTIONAL] I think having this route as `quotes` instead of `about` makes more sense because the corresponding page is supposed to be displaying quotes. 😉